### DISCUSSION TITLE
lower the charge current

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#427] [Q&A] [answered]
hello:) in the new driver there is the section where you can lower the charge current depending on the voltage in the utils.py

3,55   3,5   3,45  3,3
0      0        0      200
mine is like this, and its linear - so it is possible also to change the values in the first row? eg like this

3,45   3,4   3,3  3,1
0      2       50      200

? thx markus

**Comment by Louisvdw** [comment, marked as answer]
Yes you can do that
